intelligence-services/latest/using/index.md

```diff
@@ -45,7 +45,7 @@ Follow these steps to use the default (i.e. out-of-the-box) configuration of Int
         For example, you can add any of the default renditions:
 
         ```bash
-        aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities 
+        aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities, aiSpeechToText, webvtt, aiPiiEntities 
```

Comment:
This change does not look right. I assume copy-and-paste error adding the last three again ?

I assume you meant to add to the **Note** below instead ?